server/server.go

```diff
@@ -154,18 +155,20 @@ func corsHandler(w http.ResponseWriter, r *http.Request, params map[string]strin
 // setHeaders sets deafult headers and user defined headers
 func setHeaders(w http.ResponseWriter) {
 	// add our default CORS headers
-	for name, value := range DefaultCORSHeaders {
-		v, ok := value.(string)
-		if ok {
-			w.Header().Set(name, v)
+	for name, val := range DefaultCORSHeaders {
+		if val == "" {
+			log.Warn("default CORS header (%v) has no value")
 		}
+
+		w.Header().Set(name, val)
 	}
 
 	// set user defined headers
-	for name, value := range Headers {
-		v, ok := value.(string)
-		if ok {
-			w.Header().Set(name, v)
+	for name, val := range Headers {
+		if val == "" {
+			log.Warn("header (%v) has no value")
```

Comment:
you are missing the val, value in the Warn here.

It should be
```go
log.Warnf("header (%v) had no value",val)
```

Reply:
Good catch.